WalletWasabi/Helpers/StartupHelper.cs

```diff
@@ -0,0 +1,61 @@
+using Microsoft.Win32;
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using WalletWasabi.Logging;
+
+namespace WalletWasabi.Helpers
+{
+	public static class StartupHelper
+	{
+		public const string StartupErrorMessage = "Something went wrong while trying to make your changes.";
+		private const string KeyPath = "SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Run";
+
+		public static bool TryModifyStartupSetting(bool runOnSystemStartup)
+		{
+			if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+			{
+				string pathToExeFile = Assembly.GetExecutingAssembly().Location;
+				pathToExeFile = pathToExeFile.Remove(pathToExeFile.Length - 4);        // This part has to change if this gets released
+				pathToExeFile += ".Fluent.Desktop.exe";
```

Comment:
Personally, I would move it one level up. `WalletWasabi` C# project should not be really aware that there is some project using it and behave to fulfil its needs. At least that's a general principle.

Reply:
Can you explain this a little bit more? I'm not sure I got what you mean.

Reply:
`StartupHelper.cs` is located in `WalletWasabi` C# project. Line 21 mentions `Fluent.Desktop.exe`. `WalletWasabi` C# project is not supposed to be aware of Fluent project really. L19-L21 is basically a trick how to get `*.Fluent.Desktop.exe` path.

It makes more sense to me to just pass a path here in Fluent project https://github.com/zkSNACKs/WalletWasabi/pull/5960/files#diff-955aab30048a568fcdf2050ddba7340aabd7b559e78b1c02fd7f8875fb5b8828R62 rather than trying to compute it *somehow* in WalletWasabi project.

Reply:
Or you can probably just move your `StartupHelper` class to the Fluent project altogether. And then when it proves useful for other project, somebody else may figure out how to make the class re-usable.

Reply:
I got it now, thanks. It makes sense.

Reply:
Although we only use that `pathToExeFile` if we are on Windows platform. If we use any other OS then we won't ever use that variable. Maybe i should just move the whole class into the Fluent project.